operators/hpe-csi-operator/destinations/community-operators/current-version/manifests/hpe-csi-operator.v2.4.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,732 @@
+                  - args:
+                      - --health-probe-bind-address=:8081
+                      - --metrics-bind-address=127.0.0.1:8080
+                      - --leader-elect
+                      - --leader-election-id=hpe-csi-operator
+                    image: quay.io/datamattsson/csi-driver-operator@sha256:7f320d28a5845c2cb74bba6bcb906cea8c9b9115875594a978085bd9c48e3ae6
+                    livenessProbe:
+                      httpGet:
+                        path: /healthz
+                        port: 8081
+                      initialDelaySeconds: 15
+                      periodSeconds: 20
+                    name: manager
+                    readinessProbe:
+                      httpGet:
+                        path: /readyz
+                        port: 8081
+                      initialDelaySeconds: 5
+                      periodSeconds: 10
+                    resources:
+                      limits:
+                        cpu: 500m
+                        memory: 512Mi
+                      requests:
+                        cpu: 10m
+                        memory: 64Mi
+                    securityContext:
+                      allowPrivilegeEscalation: false
+                      capabilities:
+                        drop:
+                          - ALL
+                securityContext:
+                  runAsNonRoot: true
+                serviceAccountName: hpe-csi-operator-controller-manager
+                terminationGracePeriodSeconds: 10
+      permissions:
+        - rules:
+            - apiGroups:
+                - ""
+              resources:
+                - configmaps
+              verbs:
+                - get
+                - list
+                - watch
+                - create
+                - update
+                - patch
+                - delete
+            - apiGroups:
+                - coordination.k8s.io
+              resources:
+                - leases
+              verbs:
+                - get
+                - list
+                - watch
+                - create
+                - update
+                - patch
+                - delete
+            - apiGroups:
+                - ""
+              resources:
+                - events
+              verbs:
+                - create
+                - patch
+          serviceAccountName: hpe-csi-operator-controller-manager
+    strategy: deployment
+  installModes:
+    - supported: true
+      type: OwnNamespace
+    - supported: true
+      type: SingleNamespace
+    - supported: false
+      type: MultiNamespace
+    - supported: false
+      type: AllNamespaces
+  keywords:
+    - HPE
+    - HPE CSI
+    - HPE Storage
+    - CSI
+  links:
+    - name: Documentation
+      url: https://scod.hpedev.io/csi_driver/index.html
+  maintainers:
+    - email: dev-hi-containers@hpe.com
+      name: HPE Storage Containers Team
+  maturity: stable
+  minKubeVersion: 1.26.0
```

Comment:
I heard from Sai that one customer is having issue with minikube version. They are currently using 1.25 and they have to upgrade to 1.26 to use this operator.

Reply:
CSI driver 2.4.1 is not supported on 1.26 but I guess we have to set 1.25 to support OpenShift 4.12.